chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -719,4 +719,33 @@ Array Swapaxes(const Array& a, int8_t axis1, int8_t axis2) {
     return out;
 }
 
+Array ExpandDims(const Array& a, int8_t axis) {
+    Shape shape = a.shape();
+    Strides strides = a.strides();
+
+    auto ndim = a.ndim();
```

Comment:
Why not use `NormalizeAxis` defined in `chainerx/axes.h`?

Reply:
Forgot about that. Thanks.